packages/devextreme/js/ui/shared/accessibility.js

```diff
@@ -180,7 +180,7 @@ export function selectView(viewName, instance, event) {
             viewItemIndex = keyName === 'upArrow' ? --viewItemIndex : ++viewItemIndex;
             const viewName = viewNames[viewItemIndex];
             const viewSelectors = viewItemSelectorMap[viewName];
-            const $focusViewElement = findFocusedViewElement(viewSelectors, event.target);
+            const $focusViewElement = findFocusedViewElement(instanceTest.component.element(), viewSelectors, event.target);
```

Comment:
Compilation error here I guess

```suggestion
            const $focusViewElement = findFocusedViewElement(instance.component.element(), viewSelectors, event.target);
```